kolibri/plugins/user/assets/src/views/SignInPage.vue

```diff
@@ -2,192 +2,100 @@
-                  id="username"
-                  ref="username"
-                  v-model="username"
-                  autocomplete="username"
-                  :autofocus="!hasMultipleFacilities"
-                  :label="coreString('usernameLabel')"
-                  :invalid="usernameIsInvalid"
-                  :invalidText="usernameIsInvalidText"
-                  @blur="handleUsernameBlur"
-                  @input="showDropdown = true"
-                  @keydown="handleKeyboardNav"
-                />
-              </transition>
-              <transition name="list">
-                <div class="suggestions-wrapper">
-                  <ul
-                    v-if="simpleSignIn && suggestions.length"
-                    v-show="showDropdown"
-                    class="suggestions"
-                    :style="{ backgroundColor: $themeTokens.surface }"
-                  >
-                    <UiAutocompleteSuggestion
-                      v-for="(suggestion, i) in suggestions"
-                      :key="i"
-                      :suggestion="suggestion"
-                      :style="suggestionStyle(i)"
-                      @mousedown.native="fillUsername(suggestion)"
-                    />
-                  </ul>
-                </div>
-              </transition>
-              <transition name="textbox">
-                <KTextbox
-                  v-if="needPasswordField"
-                  id="password"
-                  ref="password"
-                  v-model="password"
-                  type="password"
-                  autocomplete="current-password"
-                  :label="coreString('passwordLabel')"
-                  :autofocus="simpleSignIn"
-                  :invalid="passwordIsInvalid"
-                  :invalidText="passwordIsInvalidText"
-                  :floatingLabel="!autoFilledByChromeAndNotEdited"
-                  @blur="passwordBlurred = true"
-                  @input="handlePasswordChanged"
-                />
-              </transition>
-              <div>
-                <KButton
-                  class="login-btn"
-                  type="submit"
-                  :text="coreString('signInLabel')"
-                  :primary="true"
-                  :disabled="busy"
-                />
-              </div>
-            </form>
-
-            <p class="create">
-              <KRouterLink
-                v-if="canSignUp"
-                :text="$tr('createAccountAction')"
-                :to="signUpPage"
-                :primary="true"
-                appearance="flat-button"
-              />
-            </p>
-            <div slot="options">
-              <component :is="component" v-for="component in loginOptions" :key="component.name" />
-            </div>
-            <p
-              v-if="showGuestAccess"
-              class="guest small-text"
+    <AuthBase>
+      <div
+        v-if="hasMultipleFacilities && selectedFacility.name"
```

Comment:
This "Sign into 'Facility name'" should be enabled for a single-facility device when the `DeviceSetting.preset` is formal/nonformal. Right?

Reply:
`selectedFacility` will only have a `name` if it's Formal/Nonformal

However using the preset makes sense - will need to see where that lives in the Store - I only looked for it on the Facility dataset properties

Reply:
That `preset` is set for the Facility. I added it to the `FacilitySerializer` and check for that here instead now.

Reply:
Actually - since this is not shown for an Informal facility - wouldn't we want a separate string altogether here in that case?

If all three facilities presets are possible on a single device - then we still want to show the user the ability to change their facility, right?

I'll raise this in the peer-sync Slack as well - just wanted to note it here.